packages/unimodules-test-core/android/src/main/java/expo/modules/test/core/ModuleController.kt

```diff
@@ -0,0 +1,34 @@
+package expo.modules.test.core
+
+import expo.modules.kotlin.ModuleHolder
+import expo.modules.kotlin.events.EventName
+
+interface ModuleController {
```

Comment:
I'm not sure about this name 🤔 
From what I can see this is not a controller that controls the module in any way, but rather some lifecycle evens dispatcher 🤔
What's the non-testing class that is actually handling these methods?
What about renaming and adding a bit of description in the comment.
I suggest `LifecycleEventsDispatcher` or something more concise, but I'm lacking in understanding the full scope of what this interface is actually doing.

Reply:
> What's the non-testing class that is actually handling these methods?

They're called in numerous places. So there isn't one abstraction. In the non-testing code, they are treated as events that can be handled by the module. 

> From what I can see this is not a controller that controls the module in any way, but rather some lifecycle evens dispatcher 🤔

That's true for now, cause I've only added support for lifecycle events, but in the future, this will be responsible for dispatching other `non-functions` (not exported) methods. That's why I've chosen the `ModuleController`.

Reply:
Ok, I'm fine with that then, but let's add a few words in the comment okay? 🤔